src/seedu/addressbook/data/tag/Tagging.java

```diff
@@ -0,0 +1,42 @@
+package seedu.addressbook.data.tag;
+
+import seedu.addressbook.data.person.Person;
+import seedu.addressbook.data.tag.Tag;
+
+/**
+ * @author user
+ * This class represent each tagging relation for every person and his tags.
+ */
+public class Tagging {
+	
+	private Person person;
+	private Tag tag;
+	private boolean isAdd; 
+	
+	/**
+	 * Construct a Tagging class when adding a tag to a person
+	 */
+	public Tagging(Person person, Tag tag, boolean isAdd){
+		this.person = person;
+		this.tag = tag;
+		this.isAdd = isAdd;
+	}
+	
+	/**
+	 * Deletes the tag of a person
+	 */
+	public void deleteTag(){
```

Comment:
You are on the right track! However, the question specify that you need to keep track of both the adding and deleting. So, it may be more useful to construct a new Tagging object, but with the isAdd set to false.